scene/2d/animated_sprite_2d.cpp

```diff
@@ -496,18 +602,28 @@ void AnimatedSprite2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_frame", "frame"), &AnimatedSprite2D::set_frame);
 	ClassDB::bind_method(D_METHOD("get_frame"), &AnimatedSprite2D::get_frame);
 
+	ClassDB::bind_method(D_METHOD("set_frame_and_progress", "frame", "progress"), &AnimatedSprite2D::set_frame_and_progress);
```

Comment:
Please move this line after `set_frame_progress` and `get_frame_progress`, to match the 3D counterpart.